snap/snapcraft.yaml

```diff
@@ -319,6 +347,8 @@ apps:
     environment:
       BINDING_PUBLISHTOPIC: events
       EDGEX_SECURITY_SECRET_STORE: "false"
+      SERVICE_CONFIGACCESSTOKENFILE: $SNAP_DATA/secrets/appservice-rules-engine/consul-token
+      REGISTRY_ACCESSTOKENFILE: $SNAP_DATA/secrets/appservice-rules-engine/consul-token
```

Comment:
These need to be `$SNAP_DATA/secrets/edgex-application-service/consul-token` to ensure that they end up in the same directory as the vault token.

Reply:
that actually is the service key for using the rule engine's profile, see https://github.com/edgexfoundry/app-service-configurable/blob/master/res/rules-engine/configuration.toml#L36  The "edgex-application-service" folder is actually too generic and it's not being used by application service.  I think we should remove that one but removing that one seems to be out of scope of this PR.

Reply:
those file-based configuration of edgex-go services are removed now, so no longer an issue here.